[DLS-4426] Upgrade frontend

nubz opened the pull request.

TomRafferty reviewed (dismissed):
All looks good to me 👍

ruthdas03 commented:
<img width="622" alt="Screenshot 2023-01-19 at 15 32 08" src="https://user-images.githubusercontent.com/86231805/213484162-1bbdd39d-d09e-4ed1-a26d-4c314bab74af.png">
Looks like the 'Start again' button is missing in the curl request page, unless we don't need that button. 

Also a minor observation on the use of appConfig, seems there are quite a few html pages that's not used and shows as a warning on startup
<img width="1386" alt="Screenshot 2023-01-19 at 15 38 09" src="https://user-images.githubusercontent.com/86231805/213485776-f587e1e8-fd56-4eda-870a-f5c15ec1cf25.png">

nubz commented:
@ruthdas03 I will remove those unused implicits if they don't break anything

> <img alt="Screenshot 2023-01-19 at 15 32 08" width="622" src="https://user-images.githubusercontent.com/86231805/213484162-1bbdd39d-d09e-4ed1-a26d-4c314bab74af.png">
> 
> Looks like the 'Start again' button is missing in the curl request page, unless we don't need that button.
> Also a minor observation on the use of appConfig, seems there are quite a few html pages that's not used and shows as a warning on startup <img alt="Screenshot 2023-01-19 at 15 38 09" width="1386" src="https://user-images.githubusercontent.com/86231805/213485776-f587e1e8-fd56-4eda-870a-f5c15ec1cf25.png">

@ruthdas03  I have fixed all those warnings and removed unused implicits - also the start again button makes 2 buttons on the page and when trying to keep them to a minimum we really ought to be looking to see if they are actually needed, I'd like to say that the back link already has the same url baked in as the old start button so we don't need it
